content/en/containers/amazon_ecs/_index.md

```diff
@@ -219,6 +219,112 @@ While it's possible to run the Agent in `awsvpc` mode, it's not the recommended
 
 Instead, run the Agent in bridge mode with port mapping to allow easier retrieval of [host IP through the metadata server][6].
 
+#### FIPS Proxy for GOVCLOUD environments.
```

Comment:
I recommend you add the site region shortcode to this PR, so that the new content only appears when the US1-FED site region is selected from the Datadog Site drop-down menu.

Reply:
done - Thanks for the rec.